### PR TITLE
fix: correcting conditional that always goes to true

### PR DESCRIPTION
**Pull request by viniciuswz**
#### What problem is this solving?

We are using this native vtex component to customize some images on department and category pages, we have a banner at the top of the page.
This banner has a non-standard behavior, basically it makes a redirect to the same page when there is no redirect url registered.
In this case, the anchor tag shouldn't even be rendered, so we went to add this conditional, but to our surprise, we saw that there was already this validation, but always set to true, that is, even without a redirect link, the image will always have a redirect.

Basically today the conditional checks an empty object, but an empty object always returns true, that is, a truthy value.

I checked and corrected this conditional.

#### How to test it?

[Workspace](https://testevinicius--zonasul.myvtex.com/admin/hortifruti)

#### Screenshots or example usage:

**Redirect url has been registered**

admin-with-link
**In the inspector element, we see the anchor tag around the image we registered**
result-with-link

**Now let's remove the redirect url inside the admin**
admin-without-link

**And as expected we don't see an anchor tag around our image**
result-without-link

#### Describe alternatives you've considered, if any.
Initially I thought that there was no such verification, I would create it myself, but when I read the code I came across that there was validation but they just forgot to validate a detail.

#### How does this PR make you feel? [:link:](https://media.giphy.com/media/KmHueA88mFABT9GkkR/giphy.gif)

my-feel

### CODE REVIEW

**Comment by viniciuswz** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [x]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by viniciuswz** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Review by igorbrasileiro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [0.12.1] - 2021-07-06
+
+### Fixed
+- prevent thruty value on condition
+
```
Comment (victorhmp) [inline CHANGELOG.md]: There's no need to manually input the version number here, the CI/CD bot will take care of it :)

```suggestion
## [Unreleased]
### Fixed
- prevent thruty value on condition

```

**Review by victorhmp — APPROVED** [review]
LGTM, just a minor detail in the CHANGELOG

**Review by icazevedo — APPROVED** [review]

**Comment by viniciuswz** [comment]
> LGTM, just a minor detail in the CHANGELOG

Thanks for sharing this information, next contributions I will use that way.

**Pull request merged**